chalice/cli/__init__.py

```diff
@@ -286,10 +286,14 @@ def package(ctx, single_file, stage, out):
 
 
 @cli.command('generate-pipeline')
+@click.option('-i', '--codebuild-image',
+              help=("Specify default codebuild image to use.  "
+                    "This option must be provided when using a python "
+                    "version besides 2.7."))
```

Comment:
Do you think we should print out that specific error message now when they try to use an unknown python with no `--codebuild-image`?